tests/test_null.c

```diff
@@ -52,6 +51,25 @@ int main(void)
 	else
 	{
 		puts("ERROR: failed to parse");
+		return 1;
 	}
-	return retval;
+
+	struct json_object *jso = json_object_new_object();
+	struct json_object *null_object = json_object_new_null();
+	if (!strcmp(json_object_get_string(null_object), "null")) {
+		puts("Got 'null' string for null object");
+	} else {
+		printf("ERROR: expected 'null' string for null object ; got '%s'", json_object_get_string(null_object));
+		json_object_put(jso);
+		json_object_put(null_object);
+		return 1;
+	}
+
+	json_object_object_add(jso, "a_null_object", null_object);
+	printf("%s\n", json_object_get_string(jso));
+
+	json_object_put(null_object);
+	puts("PASS");
```

Comment:
This should be left out, since whether it actually passed depends on whether all of the output matches what's in the .expected file.